views/elements/current_and_next.tmpl

```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
+
+	{{if .Current}}
```

Comment:
So I can't find it in the docs, but from some of the example files I've seen, it seems two spaces for the .tmpl files might be best and tabs for the .go files